II.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,8 @@
 // the input is an array of numbers and strings
 // return the average of all the numbers
-// be sure to exclude the strings
\ No newline at end of file
+// be sure to exclude the strings
+exports.average = elements => {
```

Comment:
I think the recommended way to export a function in Node is to use `module.exports = YOUR_EXPORT`

Reply:
ok.